src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -151,13 +148,7 @@ public ProcessResult updateCase(
                 if (errorMsg.isPresent()) {
                     return new ProcessResult(singletonList(errorMsg.get()), emptyList());
                 } else {
-                    var updatedExceptionRecordData = exceptionRecordFinalizer.finalizeExceptionRecord(
-                        existingCase.getData(),
-                        Long.toString(existingCase.getId()),
-                        CcdCallbackType.ATTACHING_SUPPLEMENTARY_EVIDENCE
-                    );
-
-                    return new ProcessResult(updatedExceptionRecordData);
+                    return new ProcessResult(emptyList(), emptyList());
```

Comment:
should it not use another constructor instead?

Reply:
Those are the only 2.
Caller only uses warnings and errors.